runtime/lua/vim/vimhelp.lua

```diff
@@ -2,26 +2,26 @@
 
 local M = {}
 
--- Called when editing the doc/syntax.txt file
-function M.highlight_groups()
-  local save_cursor = vim.fn.getcurpos()
-
-  local start_lnum = vim.fn.search([[\*highlight-groups\*]], 'c')
-  if start_lnum == 0 then
-    return
-  end
-  local end_lnum = vim.fn.search('^======')
-  if end_lnum == 0 then
-    return
-  end
-
+-- apply current colorscheme to lists of default highlight groups
```

Comment:
```suggestion
--- Apply current colorscheme to lists of default highlight groups
```
Why not make this part of the public documentation?

Reply:
Because it wasn't meant to be public documentation ;)

We can certainly _make_ it public if plugins want to show off their highlight groups (via a filetype autocommand)?

Reply:
Again, this will be included in the LSP hover for maintainers that are using this code.